# HG changeset patch
# User Dmitry Volyntsev <xeioex@nginx.com>
# Date 1524763304 -10800
# Node ID 4e647f0bf155f0e858a8838c216e3ba1559ff870
# Parent  1305b1701099541bcb47579ce1719550662e0d1c
Fixed unit tests exit code.

Previously, 0 was returned regardless of failures.

Patch-Source: http://hg.nginx.org/njs/rev/4e647f0bf155

diff -r 1305b1701099 -r 4e647f0bf155 njs/test/njs_unit_test.c
--- a/njs/test/njs_unit_test.c	Thu Apr 26 19:58:26 2018 +0300
+++ b/njs/test/njs_unit_test.c	Thu Apr 26 20:21:44 2018 +0300
@@ -9878,7 +9878,7 @@
         printf("njs unit tests passed\n");
     }
 
-    return NXT_OK;
+    return rc;
 }
 
 

